datalad/utils.py

```diff
@@ -21,6 +21,7 @@
 import platform
 import gc
 import glob
+import wrapt
 
 from contextlib import contextmanager
 from functools import wraps
```

Comment:
Actually ATM we don't need it. But having it provides an easy way to improve all existing decorators without the need to rewrite them. So I'd vote for keeping it.